pki/certs_test.go

```diff
@@ -157,16 +157,85 @@ func TestIssueTestCertificate(t *testing.T) {
 		return foundPoison
 	}
 	if !findCTPoison(certPair.PreCert) {
-		t.Errorf("PreCert was missing expected CT Poision Extension ID")
+		t.Errorf("PreCert was missing expected CT Poison Extension ID")
 	}
 	if findCTPoison(certPair.Cert) {
-		t.Errorf("Cert had unexpected CT Poision Extension ID")
+		t.Errorf("Cert had unexpected CT Poison Extension ID")
 	}
 }
 
 func TestIssueTestCertificateWindow(t *testing.T) {
 	issuerKey, _ := RandKey()
 	issuerCert := &x509.Certificate{}
+
+	windowStart, _ := time.Parse(time.RFC3339, "2000-01-01T00:00:00Z")
+	windowEnd, _ := time.Parse(time.RFC3339, "2001-01-01T00:00:00Z")
+
+	shortFormat := func(t time.Time) string {
+		return t.Format("2006-01-02")
+	}
+
+	testCases := []struct {
+		now, notBefore, notAfter string
+	}{
+		{now: "1995-05-05", notBefore: "1995-05-05", notAfter: "2000-12-31"},
```

Comment:
Thanks for making this nice table driven test!

I think it would be helpful (and match existing tests) if you could add an English description of each testcase in a   `Name` field and use that as the first argument to `t.Run` instead of the testcase `now`.

Alternatively, can you add a comment for each test case that describes what you're testing with the given now/notBefore/notAfter combination?